service/kvmconfig/v11/drainer_resource_set.go

```diff
@@ -1,24 +1,25 @@
-package v2
+package v11
```

Comment:
The pod resource got moved and is now v11. It does not do anything and did not change. It is just versioned together with the rest, which is cool.